grass7/imagery/i.landsat/i.landsat.qa/i.landsat.qa.html

```diff
@@ -0,0 +1,177 @@
+<br>
+
+<p>The following quality relevant conditions are represented as "single bits" 
+in the Landsat QA band:</p>
+
+<table border="1" padding="0" spacing="1">
+  <tr>
+    <td>Condition</td>
+    <td>QA Bit Position</td>
+  </tr>
+  <tr>
+    <td>Designated Fill</td>
+    <td>0</td>
+  </tr>
+  <tr>
+    <td>Terrain Occlusion / Dropped Pixels</td>
+    <td>1</td>
+  </tr>
+  <tr>
+    <td>Cloud</td>
+    <td>4</td>
+  </tr>
+</table>
+
+<p>Possible choices for the "single bits" are:</p>
+
+<table border="1" padding="0" spacing="1">
+  <tr>
+    <td>Value</td>
+	<td>Description</td>
+	<td>Bit representation</td>
+  </tr>
+  <tr>
+    <td>No</td>
+	<td>This condition does not exist</td>
+	<td>0</td>
+  </tr>
+  <tr>
+    <td>Yes</td>
+	<td>This condition exists</td>
+	<td>1</td>
+  </tr>
+</table>
+
+<p>The following quality relevant conditions are represented as "double bits" 
+in the Landsat QA band:</p>
+
+<table border="1" padding="0" spacing="1">
+  <tr>
+    <td>Condition</td>
+    <td>QA Bit Position</td>
+  </tr>
+  <tr>
+    <td>Radiometric saturation</td>
+    <td>2</td>
+  </tr>
+  <tr>
+    <td>Cloud Confidence</td>
+    <td>5-6</td>
+  </tr>
+  <tr>
+    <td>Cloud Shadow Confidence</td>
+    <td>7-8</td>
+  </tr>
+  <tr>
+    <td>Snow/Ice Confidence</td>
+    <td>9-10</td>
+  </tr>
+  <tr>
+    <td>Cirrus Confidence</td>
+    <td>11-12</td>
+  </tr>
+</table>
+
+<p>Possible choices for the "double bits" are:</p>
+
+<table border="1" padding="0" spacing="1">
+  <tr>
+    <td>Value</td>
+	<td>Description</td>
+	<td>Bit representation</td>
+  </tr>
+  <tr>
+    <td>Not Determined</td>
+	<td>Algorithm did not determine the status of this condition</td>
+	<td>00</td>
+  </tr>
+  <tr>
+    <td>Low</td>
+	<td>Algorithm has low to no confidence that this condition exists 
+	(0-33 percent confidence)</td>
+	<td>01</td>
+  </tr>
+  <tr>
+    <td>Medium</td>
+	<td>Algorithm has medium confidence that this condition exists 
+	(34-66 percent confidence)</td>
+	<td>10</td>
+  </tr>
+  <tr>
+    <td>High</td>
+	<td>Algorithm has high confidence that this condition exists 
+	(67-100 percent confidence)</td>
+	<td>11</td>
+  </tr>
+</table>
+
+<h2>NOTES</h2>
+
+The Landsat Quality Assessment band is an artificial band which represents 
+an analysis based on <a href=https://landsat.usgs.gov/documents/LDCM_CVT_ADD.pdf>
+defined algorithms</a>. The USGS provides the users with the following note on 
+how the QA band should be used:
+
+<p>"Rigorous science applications seeking to optimize the value of pixels used 
+in a study will find QA bits useful as a first level indicator of certain 
+conditions. Otherwise, users are advised that this file contains information 
+that can be easily misinterpreted and it is not recommended for general use."</p>
+
+<h2>EXAMPLE</h2>
+
+<div class="code"><pre>
+#Create a cloud mask:
+i.landsat.qa --overwrite --verbose cloud="Maybe,Yes" \
+    output=./Cloud_Mask_rules.txt
```

Comment:
to make it clearer, we could add here `collection=1 sensor="Landsat 8 OLI/TIRS"` <-- is that right?